test/extended/dr/static_pod_rollouts.go

```diff
@@ -0,0 +1,110 @@
+package dr
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	librarygov1helpers "github.com/openshift/library-go/pkg/operator/v1helpers"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+)
+
+var _ = g.Describe("[sig-etcd][Feature:DisasterRecovery][Suite:openshift/etcd/recovery][Disruptive] etcd", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("etcd-static-pod-rollouts").AsAdmin()
+
+	etcdNamespace := "openshift-etcd"
+	var initialEtcdPodCount int   // the number of etcd pods in the initial state of the cluster
+	var etcdTargetNodeName string // node from which the etcd pod is to be removed
+
+	g.BeforeEach(func() {
+		isSingleNode, err := exutil.IsSingleNode(context.Background(), oc.AdminConfigClient())
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if isSingleNode {
+			g.Skip("the test is for etcd peer communication which is not valid for single node")
+		}
+
+		etcdPods, err := e2epod.GetPods(context.Background(), oc.AdminKubeClient(), etcdNamespace, map[string]string{"app": "etcd"})
+		o.Expect(err).ToNot(o.HaveOccurred())
+		initialEtcdPodCount = len(etcdPods)
+
+		masterNodes := masterNodes(oc)
+		etcdTargetNodeName = masterNodes[0].Name
+	})
+
+	g.AfterEach(func(ctx context.Context) {
+
+		// Reset the hardware speed back to default to leave the cluster in the same state.
+		g.GinkgoT().Log("resetting the hardware speed back to default")
+		data := fmt.Sprintf(`{"spec": {"controlPlaneHardwareSpeed": ""}}`)
```

Comment:
```suggestion
		data := fmt.Sprintf(`{"spec": {"logLevel": "DEBUG"}}`)
```

let's change the log level instead, I think that's less benign than changing the etcd latencies around